hexrd/fitting/calibration/calibrator.py

```diff
@@ -48,7 +45,6 @@ def residual(self, calibration_data=None):
         the calibration class instead, in which case, calibration_data can
         be `None`.
         """
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.